src/components/themed/BuyCrypto.tsx

```diff
@@ -45,13 +43,13 @@ export const BuyCrypto = (props: Props) => {
 
   return (
     <>
-      {allowedPluginIds.includes(pluginId) && tokenId == null && (
+      {allowedPluginIds.includes(pluginId) && tokenId == null && !hideNonUkCompliantFeat && (
```

Comment:
Instead of expanding the bad old code, let's update it to use `?:`, so `!allowedPluginIds.includes(pluginId) || tokenId != null || hideNonUkCompliantFeat ? null : ...`